packages/core/src/App.ts

```diff
@@ -3,33 +3,46 @@
  * SPDX-License-Identifier: MIT
  */
 
-import pino from 'pino';
+import pino, { Level } from 'pino';
 import { ClassReflection, ClassType, reflect } from '@davinci/reflector';
+import deepmerge from 'deepmerge';
 import { Module } from './Module';
 import { mapSeries } from './lib/async-utils';
 import { coerceArray } from './lib/array-utils';
 
-const logger = pino({ name: 'app' });
-
 export interface AppOptions {
 	controllers?: ClassType[];
+	logger?: {
+		level?: Level | 'silent';
+	};
 }
 
 export class App extends Module {
+	logger = pino({ name: 'app' });
```

Comment:
Should we allow to customise the logger name?

Reply:
that'd be a good addition